pennylane/labs/resource_estimation/ops/qubit/parametric_ops_single_qubit.py

```diff
@@ -0,0 +1,623 @@
+
+    @classmethod
+    def default_resource_decomp(cls, eps=None, **kwargs) -> list[GateCount]:
+        r"""Returns a list representing the resources of the operator. Each object represents a quantum gate
+        and the number of times it occurs in the decomposition.
+
+        Resources:
+            The phase shift gate is equivalent to a Z-rotation upto some global phase,
+            as defined from the following identity:
+
+            .. math:: R_\phi(\phi) = e^{i\phi/2}R_z(\phi) = \begin{bmatrix}
+                        1 & 0 \\
+                        0 & e^{i\phi}
+                    \end{bmatrix}.
+        """
+        rz = resource_rep(ResourceRZ, {"eps": eps})
+        global_phase = resource_rep(plre.ResourceGlobalPhase)
+        return [GateCount(rz), GateCount(global_phase)]
+
+    @classmethod
+    def default_adjoint_resource_decomp(cls, eps=None) -> list[GateCount]:
+        r"""Returns a list representing the resources for the adjoint of the operator.
+
+        Resources:
+            The adjoint of a phase shift operator just changes the sign of the phase, thus
+            the resources of the adjoint operation results in the original operation.
+
+        Returns:
+            list[GateCount]: A list of GateCount objects, where each object
+                represents a specific quantum gate and the number of times it appears
+                in the decomposition.
+        """
+        return [GateCount(cls.resource_rep(eps=eps))]
+
+    @classmethod
+    def default_pow_resource_decomp(cls, pow_z, eps=None) -> list[GateCount]:
+        r"""Returns a list representing the resources for an operator raised to a power.
+
+        Args:
+            pow_z (int): the power that the operator is being raised to
+
+        Resources:
+            Taking arbitrary powers of a phase shift produces a sum of shifts.
+            The resources simplify to just one total phase shift operator.
+
+        Returns:
+            list[GateCount]: A list of GateCount objects, where each object
+                represents a specific quantum gate and the number of times it appears
+                in the decomposition.
+        """
+
+        return [GateCount(cls.resource_rep(eps=eps))]
+
+
+class ResourceRX(ResourceOperator):
+    r"""Resource class for the RX gate.
+
+    Keyword Args:
+        eps (float): error threshold for clifford plus T decomposition of this operation
+        wires (Any, Wires, optional): the wire the operation acts on
+
+    Resources:
+        A single qubit rotation gate can be approximately synthesised from Clifford and T gates. The
+        resources are approximating the gate with a series of T gates. The expected T-count is taken
+        from (the 'Simulation Results' section) `Efficient Synthesis of Universal Repeat-Until-Success
+        Circuits <https://arxiv.org/abs/1404.5320>`_. The cost is given as:
+
+        .. math:: T_{count} = \ceil(1.149 * log_{2}(\frac{1}{\epsilon}) + 9.2)
```

Comment:
This was not rendering well, can you propagate the change to the other decompositions docs
```suggestion
        .. math:: T_{count} = \lceil(1.149 * log_{2}(\frac{1}{\epsilon}) + 9.2)\rceil
```

Reply:
I think the rest are fine, this one seems to be a typo.